test/steps/ConsumerSteps.scala

```diff
@@ -12,9 +12,10 @@ class ConsumerSteps extends ScalaDsl with EN with Matchers with ConsumerMarshall
 
   private val ConsumerTokenPattern = """^[a-f0-9]{64}$""".r
 
-  And("""^the (.*) endpoint receives a POST request:$""") { (endpoint: String, json: String) =>
+  And("""^the (.*) endpoint receives a PATCH request:$""") { (endpoint: String, json: String) =>
     val response = Http(AppHost + endpoint)
       .postData(json.stripMargin)
```

Comment:
The `.postData`, then setting method to `PATCH` is a litle weird, but the lib doesn't seem to support `patchData`, so I guess this is okay 😄

Reply:
Yep, the it's the only way.